### DISCUSSION TITLE
No news from lllyasviel. ;)

### DISCUSSION THREAD

**Discussion by SunGreen777** [lllyasviel/stable-diffusion-webui-forge#2394] [General]
What are you working on now? Can you tell me about your plans?  Really looking forward to something new ) 
Looking forward to major updates
Smiles ; )

**Comment by Haoming02** [comment]
> What are you working on now?

[IC-Light](https://github.com/lllyasviel/IC-Light/discussions/109)

**Reply by SunGreen777** [reply]
> > What are you working on now?
> 
> [IC-Light](https://github.com/lllyasviel/IC-Light/discussions/109)

I saw this later, it’s interesting about Forge, I really like this UI, I want support for the new one and integration of IC-Light 2
